sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1215,10 +1215,10 @@ private void SetNameFieldsIfNull()
 
             Task<Response<BlobProperties>> GetPropertiesAsync(bool async, CancellationToken ct)
                 =>
-                client.GetPropertiesAsync(
+                client.GetPropertiesInternal(
                         accessConditions: blobAccessConditions,
-                        cancellationToken: ct
-                        );
+                        async: async,
+                        cancellationToken: ct);
```

Comment:
Thanks for fixing this part of it.  Ideally we'd also make this an `async` function and add a `ConfigureAwait(false)`.  Errors flow better: https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#prefer-asyncawait-over-directly-returning-task